crates/cxx-qt-gen/src/generator/cpp/externcxxqt.rs

```diff
@@ -32,10 +32,6 @@ pub fn generate(
             let qobject_name = type_names.lookup(&signal.qobject_ident)?;
```

Comment:
can we have a changelog entry as the API has changed a bit here?

Also does the book need updating it probably mentions the locking trait?